src/platforms/unity/troubleshooting.mdx

```diff
@@ -37,6 +37,10 @@ The `Sentry.Unity.Editor.iOS.dll` is responsible for adding native support to yo
 
 ## Build Issues
 
+### Build input file cannot be found: '**/Libraries/io.sentry.unity/Plugins/iOS/SentryNativeBridge.m'
+
+You can resolve this issue by creating a clean build or chosing **Replace** when prompted. Instead of relying on Unity to copy the `SentryNativeBridge.m` over to the generated Xcode project, starting 0.12.0, the SDK copies it manually to `**/Libraries/io.sentry.unity/SentryNativeBridge.m`.
```

Comment:
Is there an alternative solution like manually deleting the old file, so users don't need to chose Replace? They might have other changes they can't override?